presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -641,6 +717,27 @@ public void testPushdownSubfieldsAssorted()
                         "a", toSubfields("a[1]"),
                         "d", toSubfields("d.d3[5]")));
 
+        //Subfield pruning should pass-through arbitrary() function
+        assertPushdownSubfields("SELECT id, " +
+                        "arbitrary(x.a), arbitrary(x).a, arbitrary(x).d.d1, arbitrary(x.d).d1, arbitrary(x.d.d2), " +
```

Comment:
nit: perhaps, put each `arbitrary` on a separate line for readability

Reply:
Done